src/cts/src/TritonCTS.cpp

```diff
@@ -277,7 +277,20 @@ void TritonCTS::countSinksPostDbWrite(
                                 fullTree,
                                 sinks);
         } else {
-          logger_->report("Hanging buffer {}", name);
+          std::string cellType = "complex cell";
+          odb::dbInst* inst = iterm->getInst();
+          sta::Cell* masterCell = network_->dbToSta(inst->getMaster());
+          if (masterCell) {
+            sta::LibertyCell* libCell = network_->libertyCell(masterCell);
+            if (libCell) {
+              if (libCell->isInverter()) {
+                cellType = "inverter";
+              } else if (libCell->isBuffer()) {
+                cellType = "buffer";
+              }
+            }
+          }
+          logger_->report("Hanging {} '{}'", cellType, name);
```

Comment:
Is this a warning?  The message doesn't give a lot of information to the user

Reply:
This was an existing message that I modified after fixing a fatal.  Changed this to info message with ID.
 logger_->info(
              CTS, 121, "{} '{}' has unconnected output pin.", cellType, name);